### PR TITLE
[WIP] Added zh-tw translation

### PR DESCRIPTION
**Pull request by genesis331**
Adding `zh-tw` translation to TensorFlow Docs. The translation is still in progress.

I need some reviewers here.

- [X] /community/contribute
- [ ] /js
- [ ] /lite
- [X] /r1
- [ ] /tutorials
- [X] /README.md
- [X] /REVIEWERS

### CODE REVIEW

File: site/zh-tw/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+[docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
+
+Note: Please focus translation efforts on
+[TensorFlow 2](https://www.tensorflow.org) in the
+[site/en/](https://github.com/tensorflow/docs/tree/master/site/en/)
+directory. TF 1.x community docs will no longer be updated as we prepare for the
+2.0 release. See
+[the announcement](https://groups.google.com/a/tensorflow.org/d/msg/docs/vO0gQnEXcSM/YK_ybv7tBQAJ).
+
+# Simplified Chinese translation guide
```
Comment (lamberta) [inline site/zh-tw/README.md]:
```suggestion
# Traditional Chinese translation guide
```
?

Comment (genesis331) [reply site/zh-tw/README.md]: Ok thanks a lot!

**Comment by Jun-Xiang** [comment]
I can be the reviewer and help @genesis331 on the translation too.

**Comment by lamberta** [comment]
Thank you.

Community translations are now located in the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) repo (see [this message](https://groups.google.com/a/tensorflow.org/g/docs/c/GJPq7S_fw4c/m/lA1qfBX2EQAJ) for context).

I'll move over/duplicate existing pull requests as I find time. In the meantime, please feel free to create a new pull request in that repo 😀 Then you can close this PR and add a link to the new one.

**Pull request closed without merging**